autoload/codeium.vim

```diff
@@ -90,9 +90,18 @@ function! s:HandleCompletionsResult(out, status) abort
       let response = json_decode(response_text)
       let completionItems = get(response, 'completionItems', [])
 
-      let b:_codeium_completions.items = completionItems
+      let b:_codeium_completions.items = []
```

Comment:
We started internally deduplicating in the language server (as of #49), so I think this bit is no longer necessary.

Reply:
I removed deduplication from this PR